app/database/migrations/20191210114055-create-communication-channels.js

```diff
@@ -0,0 +1,27 @@
+'use strict';
+module.exports = {
+  up: (queryInterface, Sequelize) => {
+    return queryInterface.createTable('CommunicationChannels', {
+      communication_channel_id: {
```

Comment:
@admin-vizcentric ..Could you please confirm why primary key/ID of communication channel is string